.gitignore

```diff
@@ -20,4 +20,4 @@ k8s_*.txt
 /docs/tools/.sphinx/warnings.txt
 /docs/tools/.sphinx/.wordlist.dic
 /docs/tools/.sphinx/.doctrees/
-/docs/tools/.sphinx/node_modules
\ No newline at end of file
+/docs/tools/.sphinx/node_modules
```

Comment:
so you've added a newline here -- i like files with newlines at the end
thanks for having that here...